apps/projects/app/App.js

```diff
@@ -183,52 +184,53 @@ const App = () => {
             onResolve={handleResolveLocalIdentity}
             onShowLocalIdentityModal={handleShowLocalIdentityModal}
           >
-            <Header
-              primary="Projects"
-              secondary={
-                <TabAction />
-              }
-            />
-            <ErrorBoundary>
-
-              {selectedIssue
-                ? (
-                  <React.Fragment>
-                    <Bar>
-                      <BackButton onClick={() => setSelectedIssue(null)} />
-                    </Bar>
-                    <IssueDetail issue={selectedIssue} />
-                  </React.Fragment>
-                )
-                : (
-                  <React.Fragment>
-                    <Tabs
-                      items={tabs.map(t => t.name)}
-                      onChange={handleSelect}
-                      selected={activeIndex.tabIndex}
-                    />
-                    <TabComponent
-                      status={github.status}
-                      app={api}
-                      projects={repos}
-                      bountyIssues={issues}
-                      bountySettings={bountySettings}
-                      tokens={tokens}
-                      activeIndex={activeIndex}
-                      changeActiveIndex={changeActiveIndex}
-                      setSelectedIssue={setSelectedIssue}
-                      onLogin={handleGithubSignIn}
-                    />
-                  </React.Fragment>
-                )
-              }
-            </ErrorBoundary>
-
-            <PanelManager
-              activePanel={panel}
-              onClose={closePanel}
-              {...panelProps}
-            />
+            <DecoratedReposProvider>
+              <Header
+                primary="Projects"
+                secondary={
+                  <TabAction />
+                }
+              />
+              <ErrorBoundary>
+
+                {selectedIssue
+                  ? (
+                    <React.Fragment>
+                      <Bar>
+                        <BackButton onClick={() => setSelectedIssue(null)} />
+                      </Bar>
+                      <IssueDetail issue={selectedIssue} />
+                    </React.Fragment>
+                  )
+                  : (
+                    <React.Fragment>
+                      <Tabs
+                        items={tabs.map(t => t.name)}
+                        onChange={handleSelect}
+                        selected={activeIndex.tabIndex}
+                      />
+                      <TabComponent
```

Comment:
This component is throwing an error (on line 164) now after:
1. Signing into Github
2. Adding a Repo
3. Signing out of Github
4. Signing back into Github

Reply:
Ah, thanks, I'll look into it!

Reply:
Updated. I got to delete more code!